specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Dataset.json

```diff
@@ -1384,6 +1384,30 @@
       ], 
       "properties": { 
       }
+    },
+    "NetezzaTableDataset" : {
+      "x-ms-discriminator-value": "NetezzaTable",
+      "description": "Netezza dataset.", 
+      "type": "object", 
+      "allOf": [ 
+        { 
+            "$ref": "#/definitions/Dataset" 
+        } 
+      ], 
+      "properties": { 

```

Comment:
Is this empty properties expected?  See also line 1409.

Reply:
yes, it's there for future extension. Similar to existing ones, i.e., line 1157